README.md

```diff
@@ -66,6 +66,25 @@ Alternatively, you can build using provided Makefiles:
 make -f Makefile.manual
 ```
 
+## Memory issues during compilation
+
+Some issues can be encountered during compilation on low RAM computers.
+These issues are due to the high number of procedures generated by the preprocessor.
+They can be avoided by limiting the maximum rank of generated procedures, as follows with `CMake`:
```

Comment:
```suggestion
## Limiting the maximum rank of generated procedures

Stdlib's preprocessor (fypp) by default generates specific procedures for arrays of all ranks, up to rank 15.
This can result in long compilation times and, on some computers, exceeding available memory.
If you know that you won't need all 15 ranks, you can specify the maximum rank for which the specific procedures will be generated.
For example, with CMake:
```

(notice that I don't use `monospace style` for proper names like CMake, stdlib, fypp, and such. Code style should be reserved for code, IMO)